pom.xml

```diff
@@ -277,5 +277,16 @@
             <version>2.1</version>
             <scope>test</scope>
         </dependency>
+        <!-- Unused by the project. Specify versions of transitive dependencies -->
```

Comment:
we probably won't need these here forever right? Is a way we can say when we can remove these? Perhaps we add a note "these may be removed when.."